velox/dwio/parquet/tests/ParquetTpchTestBase.h

```diff
@@ -29,40 +31,45 @@
 #include "velox/parse/TypeResolver.h"
 #include "velox/tpch/gen/TpchGen.h"
 
-using namespace facebook::velox;
-using namespace facebook::velox::exec;
-using namespace facebook::velox::exec::test;
-using namespace facebook::velox::parquet;
-
 namespace facebook::velox::exec::test {
 namespace {
 
+static constexpr char const* kTpchConnectorId{"test-tpch"};
+
 class ParquetTpchTestBase : public testing::Test {
  public:
   ParquetTpchTestBase() {}
 
   // Setup a DuckDB instance for the entire suite and load TPC-H data with scale
   // factor 0.01.
-  void SetUp() {
+  static void SetUpTestSuite() {
```

Comment:
Would you move implementation to .cpp file? Same in other places.

Reply:
Got it, done.